thirdparty/github.go

```diff
@@ -344,6 +344,25 @@ func getInstallationToken(ctx context.Context, owner, repo string, opts *github.
 	return token, nil
 }
 
+// revokeInstallationToken revokes an installation token.
+func RevokeInstallationToken(ctx context.Context, token string) error {
+	caller := "RevokeInstallationToken"
+	ctx, span := tracer.Start(ctx, caller, trace.WithAttributes(
+		attribute.String(githubEndpointAttribute, caller),
+	))
+	defer span.End()
+
+	githubClient := getGithubClient(token, caller, retryConfig{retry: true})
+	_, err := githubClient.Apps.RevokeInstallationToken(ctx)
+
+	span.SetAttributes(attribute.Bool("success", err == nil))
```

Comment:
I wonder if this is ultimately adding more information, because we can query for error not being set, since that's already being added as an attribute.what's the response? Does it have a status code we can post instead?

Reply:
True! Let me include some things from the response that might help debugging